doc/source/placement-group.rst

```diff
@@ -162,7 +174,12 @@ Now, you can guarantee all gpu actors and extra_resource tasks are located on th
 because they are scheduled on a placement group with the STRICT_PACK strategy.
 
 Note that you must remove the placement group once you are finished with your application.
-Workers of actors and tasks that are scheduled on placement group will be all killed:
+Workers of actors and tasks that are scheduled on placement group will be all killed
+
+.. note::
+
+  Currently, if you lose placement group handle, there's no way to remove placement group. 
+  This will be fixed in the upcoming version.
```

Comment:
```suggestion
.. warning:: Do not lose the reference to the placement group - you will not be able to remove it.
```

Reply:
This is a temporary thing. Can I say we are working in progress to fix it?

Reply:
sure, updated above